Marlin/src/pins/pins.h

```diff
@@ -1015,6 +1016,8 @@
     #error "BOARD_LINUX_RAMPS is now BOARD_SIMULATED. Please update your configuration."
   #elif MB(BTT_MANTA_M4P_V1_0)
     #error "BOARD_BTT_MANTA_M4P_V1_0 is now BOARD_BTT_MANTA_M4P_V2_1. Please update your configuration."
+  #elif MB(BOARD_TRONXY_V10)
```

Comment:
Need to drop the `BOARD_` prefix or the `MB()` macro won’t work.

Reply:
Got it. 
Thank you, sir.